### ISSUE TITLE
Add language support

### ISSUE THREAD

**Issue by Chakamorn** [jyaif/ppl-i18n#159]
I’d like to a request language support for Thai in PewPew Live.

**Comment by SKPG-Tech** [comment]
Yeah, @jyaif is going to reply on this since I don't have access to the font generator and can't know if this is going to work, I will inform the developer on this. Thanks for askingĄ

**Comment by Chakamorn** [comment]
P.S. Just saying, I can help with the translations.

**Comment by SKPG-Tech** [comment]
It is better if you patiently wait for the developer to reply to your request, Thai is _quite_ complex with its writing system, and it might not be possible to implement. That is why you shouldn't translate the game yet because it might not be used anyway, and you would just waste your time.

**Comment by SKPG-Tech** [comment]
Also quick reminder, the developer would need to add support for even generating such letters for Thai, because they have really complex curves, meaning it could take a bit longer. Sorry if you are a bit disappointed.

**Comment by Chakamorn** [comment]
That's OK.

**Comment by Chakamorn** [comment]
I’d like to ask which file is for Thai translation?
Is it the “tha” one?

**Comment by SKPG-Tech** [comment]
> I’d like to ask which file is for Thai translation? Is it the “tha” one?

Yes it is.

**Comment by SKPG-Tech** [comment]
Hey, just checking up on how's the translating going, if you have any questions feel free to ask!

**Comment by jyaif** [comment]
Did we ever conclude that we could add Thai without adding too many characters or without having to deal with ligatures?

If we did, I don't remember the conclusion of that investigation.

**Comment by SKPG-Tech** [comment]
Oh sorry! I assumed it was okay since you added the translation file.
